.github/workflows/php-ci.yml

```diff
@@ -36,6 +36,8 @@ jobs:
 
             -   uses: eonx-com/actions-composer-install@v1
 
+            -   run: composer install
```

Comment:
If this is needed only for installing EasyQuality, then let's add `--working-dir=.quality`.

Reply:
done